### ISSUE TITLE
[BUG] errors in frameset cannot be detected

### ISSUE THREAD

**Issue by gosplit** [IBMa/equal-access#285]
**Problem:**
Accessibility errors in frame cannot be detected

**Test files:**
parent.html:
```html
<html lang="en">
<title>Frameset</title>
<frameset cols="50%,50%">
	<frame title="frame1" id="frame1" src="iframe.html">
	<frame title="frame2" id="frame2" src="iframe.html">
</frameset>
<html>
```

iframe.html:
```html
<html>
<body>
<input id="no_label_error" type="text" />
</body>
<html>
```
**Environment/Version:**
Lastest Chrome and Firefox
Equal Access Accessiblity Checker 3.1.0 with lastest ruleset

**Steps:**
Step1: open parent.html
Step2: run Accessibility Checker

**Expected result:**
Report the accessiblity violation of both `iframe.html` and `parent.html`

**Current result:**
Only `parent.html` is scanned

![image](https://user-images.githubusercontent.com/15315674/101512842-6d914480-39b6-11eb-9397-118794443c6f.png)

**Comment by tombrunet** [comment]
Note that frame and frameset are deprecated elements: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/frameset

We do support iframes.

**Comment by MHoov** [comment]
As of June 1
- [x] general rule description (SME & Dev)
- [ ] testcases written (Dev)
- [ ] rule bug (Dev)
- [x] message written (SME)
- [x] help written (SME)

**Comment by aliunwala** [comment]
Notes from triage meeting:
Possibly add <frameset> to list of deprecated elements?

**Comment by marcjohlic** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Equal-Access-61ae71f9f8b4ec001e136462/issues/IBMa/equal-access/285?planning-poker) with ZenHub @aliunwala @drjoho @mbgower @philljenkins @shunguoy @tombrunet

**Comment by philljenkins** [comment]
This may no longer be an issue since iframe support added

**Comment by tombrunet** [comment]
Chrome doesn't even support `frameset` any longer. Closing this.
